src/Api/RequestAttributesExtractor.php

```diff
@@ -29,19 +31,17 @@
      *
      * @throws RuntimeException
      *
-     * @return array
+     * @return array [$resourceClass, $collectionOperation, $itemOperation, $format]
```

Comment:
we should add an example with the `@example` tag to be more clear on what those values can be